public/member_ops.js

```diff
@@ -181,8 +204,87 @@ function handle_members(event) {
       });
     } else if (event.target === createFact) {
       // Your code goes here!
+      var dataObject = {
+        member_id: memberId4.value,
+        fact_text: factText.value,
+        likes: likes.value,
+      };
+      let headers = { "Content-Type": "application/json" };
+      let csrf_cookie = getCookie("CSRF-TOKEN");
+      if (csrf_cookie) {
+        headers["X-CSRF-Token"] = csrf_cookie;
+      }
+      fetch(`${members_path}/${memberId4.value}/facts`, {
+        method: "POST",
+        headers: headers,
+        body: JSON.stringify(dataObject),
+      }).then((response) => {
+        if (response.status === 201) {
+          response.json().then((data) => {
+            resultsDiv.innerHTML = "";
+            let parag = document.createElement("P");
+            parag.textContent = JSON.stringify(data);
+            resultsDiv.appendChild(parag);
+          });
+        } else {
+          response
+            .json()
+            .then((data) => {
+              alert(
+                `Return code ${response.status} ${
+                  response.statusText
+                } ${JSON.stringify(data)}`,
+              );
+            })
+            .catch((error) => {
+              console.log(error);
+              alert(error);
+            });
+        }
+      });
+
     } else if (event.target === updateFact) {
       // Your code goes here!
+      var dataObject = {
+        member_id: memberId5.value,
+        fact_text: factText2.value,
+        likes: likes2.value,
+      };
+      let headers = { "Content-Type": "application/json" };
+      let csrf_cookie = getCookie("CSRF-TOKEN");
+      if (csrf_cookie) {
+        headers["X-CSRF-Token"] = csrf_cookie;
+      }
+      console.log(`${members_path}/${memberId5.value}/facts/${factNumber.value}`)
+      fetch(`${members_path}/${memberId5.value}/facts/${factNumber.value}`, {
+        method: "PUT",
+        headers: headers,
+        body: JSON.stringify(dataObject),
```

Comment:
Ah, here you have that exact recommended style of line breaks for the longer pieces of code! It can make it a lot easier to get an understanding of what's happening at a glance when it's broken up this way. Great work!